# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ScriptKind.h
# Commit: c11cffaded9c
# Full Hash: c11cffaded9cfed53b04e5c7875278e9a9e55396
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r 0527d227c6b6 -r c11cffaded9c js/loader/ScriptKind.h
--- a/js/loader/ScriptKind.h	Fri Feb 18 12:45:52 2022 +0000
+++ b/js/loader/ScriptKind.h	Fri Feb 18 12:45:53 2022 +0000
@@ -4,15 +4,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef mozilla_dom_ScriptKind_h
-#define mozilla_dom_ScriptKind_h
+#ifndef js_loader_ScriptKind_h
+#define js_loader_ScriptKind_h
 
-namespace mozilla {
-namespace dom {
+namespace JS::loader {
 
 enum class ScriptKind { eClassic, eModule, eEvent };
 
-}  // namespace dom
-}  // namespace mozilla
+}  // namespace JS::loader
 
 #endif